mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,409 @@
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the data file.
+    eog : list | tuple | 'auto' | 'header'
+        Names of channels or list of indices that should be designated
+        EOG channels. If 'header', VEOG and HEOG channels assigned in the file
+        header are used. If 'auto', channel names containing 'EOG' are used.
+        Defaults to empty tuple.
+    ecg : list or tuple | 'auto'
+        Names of channels or list of indices that should be designated
+        ECG channels. If 'auto', the channel names containing 'ECG' are used.
+        Defaults to empty tuple.
+    emg : list or tuple
+        Names of channels or list of indices that should be designated
+        EMG channels. If 'auto', the channel names containing 'EMG' are used.
+        Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. Defaults to empty tuple.
+    read_blocks : bool
+        Whether to read data in blocks. This is for dealing with different
+        kinds of CNT data formats.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    Instance of RawCNT.
+    """
+    return RawCNT(input_fname, eog=eog, ecg=ecg, emg=emg, misc=misc,
+                  read_blocks=read_blocks, preload=preload, verbose=verbose)
+
+
+def _get_cnt_info(input_fname, eog, ecg, emg, misc, read_blocks):
+    """Helper for reading the cnt header."""
+    data_offset = 900  # Size of the 'SETUP' header.
+    cnt_info = dict()
+    # Reading only the fields of interest. Structure of the whole header at
+    # http://paulbourke.net/dataformats/eeg/
+    with open(input_fname, 'rb', buffering=0) as fid:
+        fid.seek(21)
+        patient_id = ''.join(np.fromfile(fid, dtype='S1', count=20))
+        patient_id = int(patient_id) if patient_id.isdigit() else 0
+        fid.seek(121)
+        patient_name = ''.join(np.fromfile(fid, dtype='S1', count=20)).split()
+        last_name = patient_name[0]
+        first_name = patient_name[-1]
+        fid.seek(2, 1)
+        sex = ''.join(np.fromfile(fid, dtype='S1', count=1))
+        if sex == 'M':
+            sex = 1
+        elif sex == 'F':
+            sex = 2
+        else:  # can be 'U'
+            sex = 0
+        hand = ''.join(np.fromfile(fid, dtype='S1', count=1))
+        if hand == 'R':
+            hand = 1
+        elif hand == 'L':
+            hand = 2
+        else:  # can be 'M' for mixed or 'U'
+            hand = 0
```

Comment:
Does anyone know how hand and sex are coded in FIFF or should these be left out? It would be nice to add them since they're there.